vars/githubPr.groovy

```diff
@@ -169,26 +169,34 @@ def getNextCommentMessage(previousCommentInfo = [:], isFinal = false) {
     ? getBuildStatusIncludingMetrics()
     : buildUtils.getBuildStatus()
 
+  def storybooksUrl = buildState.get('storybooksUrl')
+  def storybooksMessage = storybooksUrl ? "* [Storybooks Preview](${storybooksUrl})" : "* Storybooks not built"
+
   if (!isFinal) {
+    storybooksMessage = storybooksUrl ? storybooksMessage : "* Storybooks not built yet"
+
     def failuresPart = status != 'SUCCESS' ? ', with failures' : ''
     messages << """
       ## :hourglass_flowing_sand: Build in-progress${failuresPart}
       * [continuous-integration/kibana-ci/pull-request](${env.BUILD_URL})
       * Commit: ${getCommitHash()}
+      ${storybooksMessage}
       * This comment will update when the build is complete
     """
   } else if (status == 'SUCCESS') {
     messages << """
       ## :green_heart: Build Succeeded
       * [continuous-integration/kibana-ci/pull-request](${env.BUILD_URL})
       * Commit: ${getCommitHash()}
+      ${storybooksMessage}
       ${getDocsChangesLink()}
```

Comment:
The newlines that end up in the list here cause the list to render much larger, which would be ideal to work around: 

![image](https://user-images.githubusercontent.com/1329312/107283001-037a3780-6a19-11eb-94a2-c9e9739f74a3.png)

vs

![image](https://user-images.githubusercontent.com/1329312/107283016-083eeb80-6a19-11eb-9f67-7fef6086d46e.png)

Reply:
I will get this fixed. I didn't notice before because it only happens if it's not the last link in the list (e.g. when there's a flaky suite)

Reply:
Ahh, it's actually `getDocsChangesLink()` that's inserting a blank newline when it's missing. I can still get it fixed in this PR

Reply:
Yeah, my bad, makes sense. Thank you